CMakeLists.txt

```diff
@@ -0,0 +1,12 @@
+cmake_minimum_required(VERSION 3.8)
+project(boost-assert)
```

Comment:
No idea; I'm not up to date with CMake naming conventions. I was just wondering. Can we use `boost_assert` here as well, or does it have to be different?

Reply:
Unfortunately, I'm not aware of any common naming scheme for cmake libraries. I prefer to have a different name for the project than the targets, but as far as I know, you can use pretty much anything you want for a name (afaik this isn't visible to the consumer anyway). 

Other names I had considered are 

BoostAssert

Or

Boost.Assert

I wasn't sure if there is any reason to avoid dots in the name, but as I haven't used them so far in project names I didn't feel like experimenting.

Reply:
Looking at https://github.com/pdimov/boost-cmake-demo/blob/master/libs/assert/CMakeLists.txt, I've used `boost_assert` for the project name there as well. It does have the advantage of avoiding the "why is one _ and the other -" questions. :-)

Reply:
Well, they are different things (e.g. one project usually has multiple different targets anyway), so I prefer to name them differently. But I have really no problem with changing the project name to whatever you prefer